streampipes-rest-core-base/src/main/java/org/apache/streampipes/rest/core/base/impl/AbstractRestResource.java

```diff
@@ -37,16 +36,59 @@
 import org.apache.streampipes.storage.api.IVisualizationStorage;
 import org.apache.streampipes.storage.management.StorageDispatcher;
 
+import org.apache.http.HttpStatus;
 import org.apache.http.client.ClientProtocolException;
-
-import jakarta.ws.rs.core.Response;
+import org.springframework.http.ResponseEntity;
 
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.net.URLDecoder;
 
-public abstract class AbstractRestResource extends AbstractSharedRestInterface {
```

Comment:
Why do we remove abstraction here? Can we completely get rid of the `AbstractSharedRestInterface`?
If so, we can do this in a follow up task.